code/modules/fish/fish_items.dm

```diff
@@ -0,0 +1,219 @@
+	icon = 'icons/obj/seafood.dmi'
+	icon_state = "shrimp_raw"
+	filling_color = "#FF1C1C"
+
+/obj/item/weapon/reagent_containers/food/snacks/shrimp/New()
+		..()
+		desc = pick("Anyway, like I was sayin', shrimp is the fruit of the sea.", "You can barbecue it, boil it, broil it, bake it, saute it.")
+		reagents.add_reagent("NUTRIMENT", 1)
+		src.bitesize = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/feederfish
+	name = "feeder fish"
+	desc = "A tiny feeder fish. Sure doesn't look very filling..."
+	icon = 'icons/obj/seafood.dmi'
+	icon_state = "feederfish"
+	filling_color = "#FF1C1C"
+
+	/obj/item/weapon/reagent_containers/food/snacks/feederfish/New()
+		..()
+		reagents.add_reagent("NUTRIMENT", 1)
+		src.bitesize = 1
+
+/obj/item/weapon/fish
+	name = "fish"
+	desc = "A generic fish"
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "fish"
+	throwforce = 1
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+	force = 1
+	attack_verb = list("slapped", "humiliated", "hit", "rubbed")
+	hitsound = 'sound/effects/snap.ogg'
+
+/obj/item/weapon/fish/glofish
+	name = "glofish"
+	desc = "A small bio-luminescent fish. Not very bright, but at least it's pretty!"
+	icon_state = "glofish"
+
+/obj/item/weapon/fish/glofish/New()
+	..()
+	set_light(2,1,"#99FF66")
+
+/obj/item/weapon/fish/electric_eel
+	name = "electric eel"
+	desc = "An eel capable of producing a mild electric shock. Luckily it's rather weak out of water."
+	icon_state = "electric_eel"
+
+/obj/item/weapon/fish/shark
+	name = "shark"
+	desc = "Warning: Keep away from tornadoes."
+	icon_state = "shark"
+	hitsound = 'sound/weapons/bite.ogg'
+	force = 3
+
+/obj/item/weapon/fish/shark/attackby(var/obj/item/O, var/mob/user as mob)
+	if(istype(O, /obj/item/weapon/wirecutters))
+		to_chat(user, "You rip out the teeth of \the [src.name]!")
+		new /obj/item/weapon/fish/toothless_shark(get_turf(src))
+		new /obj/item/stack/teeth/shark(get_turf(src))
+		qdel(src)
+		return
+	..()
+
+/obj/item/weapon/fish/toothless_shark
+	name = "toothless shark"
+	desc = "Looks like someone ripped it's teeth out!"
+	icon_state = "shark"
+	hitsound = 'sound/effects/snap.ogg'
+
+/obj/item/stack/teeth/shark
+	name = "shark teeth"
+	desc = "A number of teeth, supposedly from a shark."
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "teeth"
+	force = 2.0
+	throwforce = 5.0
+	materials = list()
+
+/obj/item/stack/teeth/shark/New()
+	src.pixel_x = rand(-5,5)
+	src.pixel_y = rand(-5,5)
+
+/obj/item/weapon/fish/catfish
+	name = "catfish"
+	desc = "Apparently, catfish don't purr like you might have expected them to. Such a confusing name!"
+	icon_state = "catfish"
+
+/obj/item/weapon/fish/catfish/attackby(var/obj/item/O, var/mob/user as mob)
+	if(O.sharpness_flags & SHARP_BLADE)
+		to_chat(user, "You carefully clean and gut \the [src].")
+		new /obj/item/weapon/reagent_containers/food/snacks/catfishmeat(get_turf(src))
+		new /obj/item/weapon/reagent_containers/food/snacks/catfishmeat(get_turf(src))
+		qdel(src)
```

Comment:
Same as the shark attackby, make sure it's either on a surface, or is dropped if the user is holding it

Reply:
It is always dropped by the user holding it, since it creates a new obj.
Do you want me to put it in the user's hands ?